### PR TITLE
Lowering log levels.

### PR DESCRIPTION
**Pull request by vmarmol**
Reduce common logging using Kubernetes logging standards.

### CODE REVIEW

**Comment by vishh** [comment]
LGTM

File: utils/cpuload/netlink/reader.go [hunk 1]
```diff
@@ -73,6 +72,5 @@ func (self *NetlinkReader) GetCpuLoad(name string, path string) (info.LoadStats,
 	if err != nil {
 		return info.LoadStats{}, err
 	}
-	glog.V(3).Infof("Task stats for %q: %+v", path, stats)
```
Comment (rjnagal) [inline utils/cpuload/netlink/reader.go]: Can you make these V(4) for the next few weeks while we play with this code?

Comment (vmarmol) [inline utils/cpuload/netlink/reader.go]: Np, done.

**Comment by rjnagal** [comment]
LGTM, thanks.

**Pull request merged**